engine/env/spark/pom.xml

```diff
@@ -136,6 +136,22 @@
 					<groupId>org.eobjects.metamodel-extras</groupId>
 					<artifactId>MetaModel-extras-access</artifactId>
 				</exclusion>
+                <exclusion>
```

Comment:
OK, I understand that rest-client is then agnostic to DC in general. So what I imagine is a dependency/module structure like this:

 * new module: engine/DataCleaner-rest-client (no DC related dependencies)
 * new module: components/DataCleaner-remote-component (dependencies: DataCleaner-rest-client and DataCleaner-engine-core)
 * new dependency: engine/DataCleaner-xml-config would need to depend on DataCleaner-remote-components, but we can set the dependency to be optional
 * new dependency: desktop/ui would need to depend on DataCleaner-remote-components
* new dependency: monitor/api would need to depend on DataCleaner-rest-client
* new dependency: monitor/services (or maybe only monitor/ui) would need to depend on DataCleaner-remote-components

That way if you build a UI-free application that depends on DC then you dont necesarily have to bring in the remote components (which is unlikely to be relevant in such a case IMO).